skills/lark-meeting/scripts/meeting_qa_bootstrap.py

```
@@ -0,0 +1,645 @@
+    """
+    统一渲染：进行中会议（含一页 pretty 事件预览）+ 今天已结束会议（含可直接执行的最终命令）+ 未来日程。
+    ended_tokens: {meeting_id: {note_id, minute_token, note_doc_token, verbatim_doc_token}}
+    scope_hint: 不为空时在末尾追加授权提示。
+    """
+    parts = ["## 会议问答上下文", ""]
+
+    has_active = bool(active)
+    has_ended = bool(ended)
+    has_calendar = bool(calendar_events)
+    if has_active and has_ended:
+        scene = "当前有进行中会议（附第一页事件预览），同时列出今天已结束的会议"
```

Ruff (0.16.1)

[warning] String contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF001)